.github/workflows/windows/build/action.yml

```diff
@@ -0,0 +1,30 @@
+name: "Build DLL files for Windows"
+description: "Prepares the PHP build environment for the MongoDB driver"
+inputs:
+  version:
+    description: "PHP version to build for"
+    required: true
+  arch:
+    description: "The architecture to build for (x64 or x86)"
+    required: true
+  ts:
+    description: "Thread-safety (nts or ts)"
+    required: true
+outputs:
+  build-dir:
+    description: "The build directory to be used"
+    value: ${{ steps.prepare-build-env.outputs.build-dir }}
+runs:
+  using: composite
+  steps:
+    - name: Prepare build environment
+      id: prepare-build-env
```

Comment:
Should the `windows/prepare-build` action just be consolidated into this file? Unless I overlooked something, it doesn't appear to be referenced anywhere else.

Reply:
The action is used twice: once called from the build action, but also called from the `test` job in the `windows-tests` workflow. While that job reuses the build files from the `build` job in the same workflow, we still need to prepare the build environment to put the DLL file in the correct place.